src/ApaiIO/Request/GuzzleRequest.php

```diff
@@ -72,7 +73,12 @@ public function perform(OperationInterface $operation, ConfigurationInterface $c
         $request = new \GuzzleHttp\Psr7\Request('GET', $uri->withScheme($this->scheme), [
             'User-Agent' => 'ApaiIO [' . ApaiIO::VERSION . ']'
         ]);
-        $result = $this->client->send($request);
+        try {
+            $result = $this->client->send($request);
+        }
+        catch (RequestException $e) {
+            return NULL;
```

Comment:
An exception is already throw in send method. You can try/catch runOperation method not ?